### PR TITLE
Add index for docs

### PR DESCRIPTION
**Pull request by Amber1990Zhang**
Add doc index

### CODE REVIEW

**Comment by whitewum** [comment]
I think this pr docs/nebula-design-concepts.md  is mixed with #591
I hope this is not a git branch issue.

**Comment by whitewum** [comment]
index.md is OK to me

**Comment by Amber1990Zhang** [comment]
> I think this pr docs/nebula-design-concepts.md is mixed with #591
> I hope this is not a git branch issue.

Fixed @whitewum

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
+# **This is index for Docs**
+---
+
+## Quick start
+---
+
+* [**[Overview](https://github.com/vesoft-inc/nebula/blob/master/README.md)**] Help you get an overall concept of Nebula Graph
```
Comment (jude-zhu) [inline docs/index.md]: Help to Helps?

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+* [**[Query language nGQL](https://github.com/vesoft-inc/nebula/blob/master/docs/nGQL-tutorial.md)**] Tutorial of Nebula Graph's query language nGQL
+
+
+
+## User manual
+---
+
+* Data types
+
+  * [Numeric types](https://github.com/vesoft-inc/nebula/blob/master/docs/manual-EN/data-types/numeric-types.md)
```
Comment (jude-zhu) [inline docs/index.md]: I would prefer capitalizing every significant word

**Comment by whitewum** [comment]
close #830

**Comment by whitewum** [comment]
BTW, I prefer to follow #188

**Comment by Amber1990Zhang** [comment]
> BTW, I prefer to follow #188

Got it!

File: docs/index.md (hunk 1 above)
Comment (Amber1990Zhang) [reply docs/index.md]: > An overall concept of Nebula Graph

File: docs/index.md (hunk 2 above)
Comment (Amber1990Zhang) [reply docs/index.md]: All fix

**Comment by jude-zhu** [comment]
Jenkins go

**Pull request merged**